ml_core/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""MLonCode research playground."""
+import pkg_resources
+
+pkg_resources.declare_namespace(__name__)
```

Comment:
We should do it only in case we share the same namespace.
I suggest doing so (see review summary message), but in case we will not, we should delete this line

Reply:
Is it ok if I leave this there for now till Vadim answers, or should I remove it and add it later if needed?

Reply:
yes, let's wait for Vadim's input.